proto/bootz.proto

```diff
@@ -34,7 +34,6 @@ service Bootstrap {
   // This RPC returns the data required to put the device in a known state
   // (i.e. OS, bootloader password, etc) and applies an initial device
   // configuration.
-  // The device must use its IDevID certificate in the gRPC TLS handshake.
```

Comment:
Why is this removed? This is still a requirement for the legacy GetBootstrapData and ReportStatus RPCs

Reply:
+1